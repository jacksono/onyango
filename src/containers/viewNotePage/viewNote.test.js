import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import ViewNote from './viewNote';
/* eslint no-undef:0 */
test('View note page renders correctly', () => {
  const tree = renderer
    .create(
      <MuiThemeProvider>
        <Router>
          <ViewNote history={{}} match={{ params: {} }} />
        </Router>
      </MuiThemeProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
