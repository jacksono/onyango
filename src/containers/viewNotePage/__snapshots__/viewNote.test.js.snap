// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View note page renders correctly 1`] = `
<div
  className="page"
>
  <div>
    <div
      style={
        Object {
          "float": "right",
        }
      }
    >
      <button
        onClick={[Function]}
        type="button"
      >
        Edit
      </button>
      <button
        onClick={[Function]}
        style={
          Object {
            "marginLeft": "10px",
          }
        }
        type="button"
      >
        Delete
      </button>
    </div>
    <h1>
      
    </h1>
    <span
      style={
        Object {
          "float": "right",
          "marginTop": "-20px",
        }
      }
    >
      Written on Invalid date
    </span>
    <hr
      style={
        Object {
          "backgroundColor": "#e0e0e0",
          "border": "none",
          "height": 1,
          "margin": 0,
          "marginLeft": 0,
          "marginTop": -1,
          "muiPrepared": true,
        }
      }
    />
    <p>
      
    </p>
  </div>
</div>
`;
